pytest-backend/exasol/pytest_backend/__init__.py

```diff
@@ -157,13 +157,14 @@ def backend_aware_onprem_database_async(
 @pytest.fixture(scope="session")
 def backend_aware_onprem_database(
     backend_aware_onprem_database_async,
-) -> EnvironmentInfo | None:
+) -> Iterable[EnvironmentInfo | None]:
```

Comment:
Hm, not sure if a Generator is Iterable, I thought it is an iterator. However, it looks like both are ok.

> Simple generators that only ever yield values can also be annotated as having a return type of either Iterable[YieldType] or Iterator[YieldType]:

https://docs.python.org/3/library/typing.html#annotating-generators-and-coroutines